src/performance-analysis-spec/specific-hardware-and-software.md

```diff
@@ -49,7 +49,15 @@ limitations](../performance-methodology/morello-microarchitectural-limitations.m
 The data-dependency fix avoids undesirable (and likely unnecessary in more
 recent baseline microarchitectures) stalls on capability stores.
 
-## P128 Forced GOT (w/ data-dependency fix)
+## Benchmark ABI (w/ data-dependency fix) (w/ larger store queue)
+
+This configuration is based on the Benchmark ABI configuration, run with a
+further modified microarchitecture that also tunes the store queue size to
+address the queue pressure issue described in [Morello microarchitectural
+limitations](../performance-methodology/morello-microarchitectural-limitations.md),
```

Comment:
Should this link be to the .html file rather than .md?

Reply:
mdBook will convert them

Reply:
OK, sounds good.